pkg/common/model/item.go

```diff
@@ -0,0 +1,15 @@
+package model
+
+// Item represents the API response structure from HackerNew for an item as used for data storage
+type Item struct {
+	ID        int    `bson:"id" json:"id"`
+	Type      string `bson:"type" json:"type"`
+	Text      string `bson:"text" json:"text"`
+	URL       string `bson:"url" json:"url"`
+	Score     int    `bson:"score" json:"score"`
+	Title     string `bson:"title" json:"title"`
+	Time      int64  `bson:"time" json:"time"`
+	CreatedBy string `bson:"by" json:"by"`
+	Dead      bool   `bson:"dead" json:"dead"`
```

Comment:
Whats dead?

Reply:
It's one of the fields returned from HackerNews API